config_cmd.go

```diff
@@ -39,13 +39,13 @@ var configCmd = &cobra.Command{
 
 		c, err := editor.Cmd("Glow", configFile)
 		if err != nil {
-			return err
+			return fmt.Errorf("glow: unable to set config file: %w", err)
```

Comment:
You are wrapping all error, it's great!

But you are adding the "glow: " prefix everywhere.

It might lead to error being reported like this

"glow: foo bar: glow quz ...: glow: ..."

I think you should consider removing all the "glow: " prefix, and adding it only in a few place in main.go

Like when calling Execute in RunE, if you have an error, you wrap it to include "glow: "

You might have to do it also when validating the config in PersitantPreRunE

What do you think about it?

Reply:
That's a good point, and one of the reasons I think enforcing wraps in 100% os cases does not necessarily makes sense.

I removed the `glow:` prefix for now.